.circleci/template.yml

```
@@ -10,6 +10,17 @@ parameters:
     default: false
 
 commands:
+  halt_if_short_flow:
+    steps:
+      - run:
+          name: Check Commit Message
```

```suggestion
          name: ⭐ Short flow breakpoint - Check if the build should continue
```

Let's describe "what it checks" instead of "what it does". The latter can be read from the script.

---

The _star_ symbol is used on flow charts and Notion. Hence, I would use it as a common symbol when explaining different formats.